dftimewolf/lib/utils.py

```diff
@@ -51,43 +52,12 @@ def import_args_from_dict(value, args, config):
         else:
           value = actual_param
   elif isinstance(value, list):
-    return [import_args_from_dict(item, args, config) for item in value]
+    return [ImportArgsFromDict(item, args, config) for item in value]
   elif isinstance(value, dict):
     return {
-        key: import_args_from_dict(val, args, config)
+        key: ImportArgsFromDict(val, args, config)
         for key, val in value.items()
     }
   elif isinstance(value, tuple):
-    return tuple(import_args_from_dict(val, args, config) for val in value)
-  return value
-
-
-def check_placeholders(value):
```

Comment:
Moved this into tests `_CheckPlaceholders`

Reply:
Ack